utils/elab_plugin.py

```diff
@@ -34,8 +34,9 @@ def setup(self, calling_class):
             try:
                 self.elab.updateMetadataUrl(group=config['group'], experiment=config['experiment'])
             except ValueError as exception:
-                logging.error("ELN groupID %s or experimentID %s not set or invalid. (%s)",
```

Comment:
For consistency's sake, I think `exception` should be `error` (occurs several more times below).

Reply:
Got it.